### PR TITLE
[MIG] l10n_ar_currency_update: Migration to 16.0

### PR DESCRIPTION
**Pull request by mem-adhoc**

### CODE REVIEW

**Comment by roboadhoc** [comment]
[Pull request status dashboard](https://runbot.adhoc.com.ar/ingadhoc/odoo-argentina-ee/pull/160).

File: l10n_ar_currency_update/wizards/res_config_settings_views.xml [hunk 1]
```diff
@@ -24,7 +24,7 @@
             </label>
 
             <button name="update_currency_rates_manually" position="attributes">
-                <attribute name="attrs">{'invisible': [('currency_interval_unit', '!=', 'manually')]}</attribute>
+                <attribute name="attrs {'invisible': [('currency_interval_unit', '!=', 'manually')]}"></attribute>
```
Comment (pablohmontenegro) [inline l10n_ar_currency_update/wizards/res_config_settings_views.xml]: @mem-adhoc Maxi, como va? habría que corregir esto

**Comment by roboadhoc** [comment]
I didn't know about this PR and had to retrieve its information, you may have to re-approve it as I didn't see previous commands.

**Comment by roboadhoc** [comment]
[Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/odoo-argentina-ee/pull/160).

**Comment by zaoral** [comment]
@roboadhoc r+

**Comment by roboadhoc** [comment]
@zaoral you may want to rebuild or fix this PR as it has failed CI.

**Comment by pablohmontenegro** [comment]
@roboadhoc r+

**Pull request closed without merging**